server/src/views/frame.js

```diff
@@ -222,6 +222,7 @@ class Frame extends React.Component {
 
   clickFullPageButton(e) {
     e.preventDefault();
+    history.pushState({state: "frame"}, "", "#frame");
```

Comment:
but that will still cause the browser to jump to the frame, and then our code will scroll it back a bit. too janky.